CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+* New `maximum_speed` parameter to the driving profiles of the directions API to specify speed limit above a certain threshold set in the config file.
```

Comment:
Now when I read it again my original wording sounds a bit clunky to me.  Maybe change
>to specify speed limit above a certain

to
>for specifying a speed limit above the

Or come up with something better :)